lib/node_modules/@stdlib/stats/base/snanmeanwd/test/test.snanmeanwd.native.js

```diff
@@ -197,15 +195,13 @@ tape( 'the function supports a `stride` parameter', opts, function test( t ) {
 		NaN   // 4
 	]);
 
-	N = floor( x.length / 2 );
-	v = snanmeanwd( N, x, 2 );
+	v = snanmeanwd( 4, x, 2 );
```

Comment:
```suggestion
	v = snanmeanwd( 5, x, 2 );
```
see commented indexes for an indication of how the array is being traversed